cmake/sysbuild/sdp.cmake

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+# Update FLPR image KConfig based on main app configuration.
+#
+# Usage:
+#   flpr_egpio_update_kconfig()
+#
+function(flpr_egpio_update_kconfig)
+
```

Comment:
```suggestion
```

Reply:
@jaz1-nordic could you change it in your PR (#16592)? I can remove it here, but I think it would be better to remove it in the commit, that it was added in.